chef/cookbooks/ceph/recipes/radosgw.rb

```diff
@@ -68,6 +49,16 @@
   subscribes :restart, "template[/etc/ceph/ceph.conf]"
 end
 
+# In the systemd case, need extra targets enabled
+service "ceph-radosgw.target" do
+  action :enable
+  only_if { File.exist?("/usr/lib/systemd/system/ceph-radosgw.target") }
```

Comment:
mon.rb and osd.rb use `if service_type == "systemd"`, and `service_type` comes from `node["ceph"]["mon"]["init_style"]` or `node["ceph"]["osd"]["init_style"]`. There's no such equivalent attribute for rwg, so I went with `only_if` on the unit file instead.